redhat-cop/uncontained.io
# Migrating openshift and the org

#### What is this PR About?
Migrating openshift and the organization

#### How should we test or review this PR?

```
npm install
npm start
```

Read through doc, make sure all content looks good, metadata, and images are present, etc.

#### Is there a relevant Trello card or Github issue open for this?
n/a

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----


-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**
